includes/Modules/Sign_In_With_Google.php

```diff
@@ -57,6 +100,21 @@ public function register() {
 		add_action( 'login_form', $this->get_method_proxy( 'render_signin_button' ) );
 	}
 
+	/**
+	 * Login user.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param WP_User $user WordPress user object.
+	 */
+	protected function login_user( WP_User $user ) {
+		wp_set_current_user( $user->ID, $user->user_login );
+		wp_set_auth_cookie( $user->ID );
+		do_action( 'wp_login', $user->user_login, $user );
+		wp_safe_redirect( admin_url() );
+		exit;
```

Comment:
Do not exit here, just return and exit later after this function is being called. In this case, it will be more obvious what happens in the code. Or, we should rename this function to be `login_user_and_exit`.

Reply:
I renamed the method so that we can keep our login code more close to the [WP login code](https://github.com/WordPress/WordPress/blob/fd6357ae7bde138ecfd7b24969ed19d013d9b6d3/wp-login.php#L1411-L1423).